.npmignore

```diff
@@ -6,7 +6,6 @@ demos1
 TODO.md
 test
 testlib
-src
```

Comment:
Lets keep the src for now.  It is necessary to deploy docs.
I will clean it later